### ISSUE TITLE
Removal of VoiceSettingsSlider breaks AudioThemes3d add-on

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#9824]
### Steps to reproduce:
Running NVDA 2019.2beta2:
* Install the audioThemes3d add-on from https://addons.nvda-project.org/addons/AudioThemes3D.en.html
* Restart NVDA

### Actual behavior:
The audioThemes add-on fails to load due to the following traceback:
```
Traceback (most recent call last): 
 File "globalPluginHandler.pyo", line 22, in listPlugins 
 File "C:\nvdaclean\userConfig\addons\AudioThemes3D\globalPlugins\audioThemes\__init__.py", line 23, in <module> 
 File "C:\nvdaclean\userConfig\addons\AudioThemes3D\globalPlugins\audioThemes\dialogs\manage_dg.py", line 17, in <module> 
ImportError: cannot import name VoiceSettingsSlider 
```

### Expected behavior:
The add-on should load. and work the same as in NVDA 2019.1.1
 
#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2019.2beta2

### Further info
This broke with the merging of pr #8214, where synth and braille GUI settings were abstracted.

It should be easy enough to replace VoiceSettingsSlider back in gui/settingsDialogs.py, but perhaps add a deprication warning when it is constructed.

This seems to be the only thing from gui/settingsDialogs.py that audioThemes3d relied upon. However other add-ons could have been using other things.

The easiest thing to do here would be to just up the aadd-on api version for 2019.2 so that all add-ons need to be checked and re-versioned. However, if it is as simple as putting back one or more of these classes and marking them as deprecated then we should consider this option.
Thoughts @leonardder?

**Comment by codeofdusk** [comment]
For whatever it's worth, I vote against an add-on version bump for 2019.2 (add-ons will already break once this year for threshold and more than that could pose a challenge for users, especially those less technically inclined). Could you add in deprecation warnings and then just remove the fallback classes in threshold?

**Comment by michaelDCurran** [comment]
The other option is to also revert #8214 and instead merge that pr into 
Threshold.

**Comment by leonardder** [comment]
This is interesting. I recall writing a patch for this, but it seems that never got through. I can dig it up and file a pr against beta. In any case, I'd vote for the deprecation warning option.
Having said that, audio themes seems pretty unmaintained, I've seen that add-on use more deprecated code.

**Comment by josephsl** [comment]
Hi, last time I brought this up on add-ons list, someone pointed out another person was working on making this add-on compatible. I think it would be best to go with deprecation warning or leave it up to the new add-on maintainer to invest in updating this add-on. Thanks.

**Comment by leonardder** [comment]
Fixed in #9826
